ports/py-qscintilla/portfile.cmake

```diff
@@ -15,20 +15,26 @@ vcpkg_extract_source_archive(
 
 file(RENAME "${SOURCE_PATH}/Python/pyproject-qt5.toml" "${SOURCE_PATH}/Python/pyproject.toml")
 
+if(VCPKG_TARGET_IS_WINDOWS)
+  set(PY_LIB_DIR "tools/python3/Lib")
```

Comment:
Questions:

 - why is this different from https://github.com/open-vcpkg/python-registry/pull/32/files#diff-306c19cfe05dc8020e1a5027728c4efe1e8333a5fb2798215db7cd7a908021deR173
 - Should this be exported from python3/vcpkg-port-config.cmake

Reply:
there is probably a cmake variable in gdal somewhere controlling the install location of python stuff.